visivo/command_line.py

```diff
@@ -39,6 +41,7 @@ def safe_visivo():
             raise e
         click.echo("An unexpected error has occurred")
         click.echo(e)
+        exit(1)
```

Comment:
Wasn't actually failing because of this...

Reply:
Ahhh, got it. Nice find